app/interactors/file_attachments/update_interactor.rb

```diff
@@ -66,6 +64,27 @@ def update_preview
   end
 
   def attachment_params
-    params.require(:file_attachment).permit(:isbn, :unique_reference)
+    raw_params = params.require(:file_attachment)
+    update_params = {}
+    update_params[:isbn] = raw_params[:isbn]
+    update_params[:unique_reference] = raw_params[:unique_reference]
+    update_params[:official_document_type] = official_document_type(raw_params)
+    update_params[:paper_number] = paper_number(raw_params)
+    update_params
+  end
+
+  def paper_number(params)
+    case params[:official_document_type]
+    when "command_paper" then params[:command_paper_number]
+    when "act_paper" then params[:act_paper_number]
+    end
+  end
+
+  def official_document_type(params)
+    case params[:official_document_type]
+    when "command_paper", "unnumbered_command_paper" then "command_paper"
+    when "act_paper", "unnumbered_act_paper" then "act_paper"
+    when "unofficial" then "unofficial"
+    end
```

Comment:
I originally thought this too. This is deliberately 'nil' because we want to keep the choice 'unselected' until the user actually makes a selection. We will add a requirements check to make the `else` irrelevant in later commits, as well.